cypress/integration/common/steps.js

```diff
@@ -550,3 +570,11 @@ Then("I see only one post with the title {string}", title => {
     .should("have.length", 1);
   cy.get(".main-container").contains(".post-link", title);
 });
+
+Then("they should not see the comment from", () => {
+  cy.get(".ds-card-footer").children().should('not.have.class', 'comment-form')
+})
+
+Then("they should see a text explaining commenting is not possible", () => {
+  cy.get('.ds-placeholder').should('contain', "Commenting is not possible at this time on this post.")
```

Comment:
That's quite important to avoid false negatives :+1: I told my students yesterday that they should look for something existing in the test before looking for something that does not exist, like in the step definition above :point_up: